bin/pycbc_multi_inspiral

```diff
@@ -236,14 +236,22 @@ parser.add_argument(
     help="Triggers with coincident/coherent SNR below this "
     "value will be discarded.",
 )
+parser.add_argument(
+    "--nifo-sngl-snr-threshold",
+    action="store",
+    type=int,
+    default=2,
+    help="Number of detectors requiring a minimum SNR value (specified "
+    "with --sngl-snr-threshold) value for trigger generation (default: 2).",
```

Comment:
I suggest "How many detectors must pass the --sngl-snr-threshold cut in order to generate a trigger (default 2)."